yb-voyager/cmd/live_migration.go

```diff
@@ -349,6 +369,10 @@ func processEvents(chanNo int, evChan chan *tgtdb.Event, lastAppliedVsn int64, d
 			utils.ErrExit("error executing batch on channel %v: %v", chanNo, err)
 		}
 		statsReporter.BatchImported(eventBatch.EventCounts.NumInserts, eventBatch.EventCounts.NumUpdates, eventBatch.EventCounts.NumDeletes)
+		if callhome.SendDiagnostics {
```

Comment:
Since these are already available in statsReporter, do we need to store them again here in different variables? During callhome payload generation, can we not directly access statsReporter?

Reply:
yes we can